filemanager.c

```diff
@@ -0,0 +1,341 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "helpers.h"
+#include "books.h"
+#include "users.h"
+#include "transactions.h"
+
+
+typedef struct Record {
+    char* data;
+    struct Record* next;
+}Record;
+
+Record* book_record_head = NULL;
+Record* user_record_head = NULL;
+Record* book_copy_record_head = NULL;
+Record* transaction_record_head = NULL;
+
```

Comment:
string root_path = ":\\Users\\annie\\CLionProjects\\Library-Management-System\\library-management-system"

const char* book_file =. root_path + "books.txt";
Kindly try this one